homework/02LessonTwo.js

```diff
@@ -84,7 +93,9 @@ function lessThanOrEqualTo(parameter1, parameter2) {
 // complete isEven() by returning a boolean expression
 
 function isEven(number) {
+   return number % 2;
 }
```

Comment:
number % 2 returns the remainder. So if it is divisible by 2, it will return 0. If not, it will return the remainder.

```javascript
if (number % 2 == 0) { return true; } 
else { return false; }
```